connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -301,6 +302,7 @@
     @Setter
     private boolean waitingForStatistics = false;
 
+    private final Map<String, Advancement> storedAdvancements = new HashMap<>();
```

Comment:
Please add a JavaDoc style comment to this and an empty line below it.

Reply:
I have no idea how to do JavaDoc comments